install_config/install/prerequisites.adoc

```diff
@@ -119,23 +119,36 @@ this during or after installation.
 The following apply to production environments. Test or sample environments will
 function with the minimum requirements.
 
-*Master Hosts*
-
+Master Hosts::
 In a highly-available {product-title} cluster with external etcd, a master host
 should have 1 CPU core and 2.5 GB of memory, on top of the defaults in the table
 above, for each 1000 pods. Therefore, the recommended size of master host in an
-{product-title} cluster of 2000 pods would be 20 CPU cores and 50 GB of RAM, as
+{product-title} cluster of 2000 pods would be 2 CPU cores and 5 GB of RAM, as
 well as the minimum requirements for a master host of 2 CPU cores and 8 GB of
 RAM.
 
-*Node Hosts*
-
+Node Hosts::
 The size of a node host depends on the expected size of its workload. As an
 {product-title} cluster administrator, you will need to calculate the expected
 workload, then add about 10% for overhead. For production environments, allocate
 enough resources so that node host failure does not affect your maximum
 capacity.
 
+Use the above with the following table to plan the maximum loads for nodes, pods, and usage:
+
+[cols="4,2",options="header"]
+|===
+|Host |Sizing Recommendation
+
+|Maximum nodes per cluster |300
+
+|Maximum pods per nodes |110
+
+|Master host resource usage at max pods |0.25 vCPU and 175 MB
```

Comment:
@timothysc I can't tell what you mean. You mean both the master and node resource usage? Just the master usage? The entire table?